corona-warn-app/cwa-app-android
# Update info screen for statistics (EXPOSUREAPP-9837)

## Whats new

Updated info screen for statistics

<img width="219" alt="Screenshot 2021-10-12 at 17 38 38" src="https://user-images.githubusercontent.com/64849422/136978116-c4540ed8-8f06-4cc0-b2a0-aa24a3948b9c.png">


## Jira
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9449)
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9837)

## How to test
1) Press on info button on any card
2) Check text and margins

**Note or testers**: kindly check not only changed that marked with magenta in Figma, but also full screen. There are multiple changes in margins and additional texts (that somehow were left between chairs in previous sprints). Thank you.
